pluto/examples/vite.config.ts

```diff
@@ -7,10 +7,18 @@
 // License, use of this software will be governed by the Apache License, Version 2.0,
 // included in the file licenses/APL.txt.
 
-import { defineConfig } from 'vite'
-import react from '@vitejs/plugin-react'
+import react from "@vitejs/plugin-react";
+import * as path from "path";
+import { defineConfig } from "vite";
 
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
-})
+  resolve: {
+    // alias: {
+    //   "@synnaxlabs/pluto/dist": path.resolve(__dirname, "../dist"),
+    //   "@synnaxlabs/pluto": path.resolve(__dirname, "../src"),
+    //   "@synnaxlabs/drift/dist": path.resolve(__dirname, "../dist"),
+    // },
```

Comment:
do we still need this commented out code

Reply:
It should be commented back in